### DISCUSSION TITLE
Upgrade HSQLDB from 2.5.0 to 2.7

### DISCUSSION THREAD

**Discussion by shutterbug79** [kagemomiji/airsonic-advanced#309] [Q&A]
Anyone who can advice on how to upgrade HSQLDB from 2.5.0 to 2.7;

Airsonic Advance: Edge Release 11.0.0-SNAPSHOT.20221224143241 - Working
OS: Ubuntu 22.04.1 LTS
Java: openjdk-17-jre
Stand-alone WAR

Having issue to get my database upgraded to version required 2.7.
The stated snapshot Edge Release 11.0.0-SNAPSHOT.20221224143241 works fine, but going beyond 11.0 (for example the latest Snap, Edge Release 11.1.3-SNAPSHOT.20231022132225 generates: 

"WARN --- o.a.p.u.LegacyHsqlMigrationUtil          : HSQLDB database upgrade needed, from version 2.5.0 to 2.7"

Thanks in advance

**Comment by kagemomiji** [comment]
@shutterbug79
This issue may be helpful for you. 
https://github.com/kagemomiji/airsonic-advanced/issues/151
